### PR TITLE
Made the code a bit more compact, while still Pythonic

### PR DESCRIPTION
**Pull request by Rembane**
I have no idea if this actually works, but it's pretty. :D

### CODE REVIEW

File: converter.py [hunk 1]
```diff
@@ -126,16 +126,13 @@ def page_predicate(page: str) -> bool:
...
-    pages = (page.split(".", 1) for page in sorted(os.listdir(directory)[offset:]) if page_predicate(page))
-
-    for (page_ns, page_name) in pages:
+   
+    for page in filter(page_predicate, sorted(os.listdir(directory)[offset:])):
+        (page_ns, page_name) = page.split(".", 1)
         if not os.path.isfile(f"converted/{page_ns}/{page_name}"):
             with open(f'converted/{page_ns}/{page_name}', 'w') as f:
-                f.write(json.dumps(findRevisions(session, directory+page_ns+"."+page_name),
+                f.write(json.dumps(findRevisions(session, '{}{}.{}'.format(directory, page_ns, page_name)),
```
Comment (Jassob) [inline converter.py]: Ey, @Rembane, what is happening here?! ❓

Comment (Jassob) [reply converter.py]: Nvm, it's some kind of string interpolation, right?

**Review by Jassob — APPROVED** [review]
Nice! :D

File: converter.py (hunk 1 above)
Comment (Rembane) [reply converter.py]: It is indeed, the more verbose kind. :)

**Pull request merged**